smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/events/MessageMarshallableGenerator.kt

```diff
@@ -63,11 +64,22 @@ class MessageMarshallableGenerator(
                                 val unbound = variant.members().filterNot {
                                     it.hasTrait<EventHeaderTrait>() || it.hasTrait<EventPayloadTrait>()
                                 }
+                                if (unbound.isNotEmpty() || eventHeaderBindings.isNotEmpty() || eventPayloadBinding != null) {
+                                    write("case .\$L(let value):", memberName)
+                                } else {
+                                    writer.write("case .\$L:", memberName)
```

Comment:
nit: `writer.apply` in enclosing block removes need for `writer.` here.

Reply:
Fixed